src/backend/src/routes/projects.routes.ts

```diff
@@ -47,5 +47,11 @@ projectRouter.post(
 projectRouter.post('/:wbsNum/set-team', nonEmptyString(body('teamId')), validateInputs, ProjectsController.setProjectTeam);
 projectRouter.delete('/:wbsNum/delete', ProjectsController.deleteProject);
 projectRouter.post('/:wbsNum/favorite', ProjectsController.toggleFavorite);
+projectRouter.post(
```

Comment:
can you add a BOM section like Ji-min does in this PR: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/1646/files